src/main/resources/swagger/GA4GH.proto

```diff
@@ -0,0 +1,294 @@
+syntax = "proto3";
```

Comment:
Let's get together some documentation on how this is generated and what caveats one should look out for when editing the openAPI version

Reply:
Created a readme